web/src/app/admin/AdminLimits.tsx

```diff
@@ -1,16 +1,15 @@
+import { useQuery, gql } from '@apollo/client'
```

Comment:
One thing of note, could you have your script make it so this isn't being placed above our `React` imports on L1?

Reply:
We do have an old codemod to sort imports you can use as an example of how it can be achieved inside the `codemods` directory.

It might be easier than manually moving them all

Reply:
I would say we should try and retain `React` on line 1 since that is one of the things we currently do as `eslint` says, and we can re-run our `jsimports` codemod when we restructure our folder coming up.

It looks like it happens when there's no `@apollo/client` present. Perhaps we can default to replacing one of the imports with the new name instead (i.e. `'@apollo/react-hooks'` -> `'@apollo/client'` in place).

Or maybe we can just insert on L2/after L1 instead? It doesn't necessarily have to be a pretty solution to get our codebase where we want it